- Liedl.csv
  - [[Double- to Single-Strand Transition Induces Forces and Motion in DNA Origami Nanostructures]]
    - Double- to Single-Strand Transition Induces Forces and Motion in DNA Origami Nanostructures
    source_file:: Liedl.csv
    Authors:: Gur, FN; Kempter, S; Schueder, F; Sikeler, C; Urban, MJ; Jungmann, R; Nickels, PC; Liedl, T
    Source_Title:: ADVANCED MATERIALS
    IF:: 30.849
    Document_Type:: Article
    Abstract:: The design of dynamic, reconfigurable devices is crucial for the bottom-up construction of artificial biological systems. DNA can be used as an engineering material for the de-novo design of such dynamic devices. A self-assembled DNA origami switch is presented that uses the transition from double- to single-stranded DNA and vice versa to create and annihilate an entropic force that drives a reversible conformational change inside the switch. It is distinctively demonstrated that a DNA single-strand that is extended with 0.34 nm per nucleotide - the extension this very strand has in the double-stranded configuration - exerts a contractive force on its ends leading to large-scale motion. The operation of this type of switch is demonstrated via transmission electron microscopy, DNA-PAINT super-resolution microscopy and darkfield microscopy. The work illustrates the intricate and sometimes counter-intuitive forces that act in nanoscale physical systems that operate in fluids.
    Publication_Year:: 2021
    DOI:: https://doi.org/10.1002/adma.202101986
  - [[Long- and short-ranged chiral interactions in DNA-assembled plasmonic chains]]
    - Long- and short-ranged chiral interactions in DNA-assembled plasmonic chains
    source_file:: Liedl.csv
    Authors:: Martens, K; Binkowski, F; Nguyen, L; Hu, L; Govorov, AO; Burger, S; Liedl, T
    Source_Title:: NATURE COMMUNICATIONS
    IF:: 14.919
    Document_Type:: Article
    Abstract:: Circular dichroism (CD) has long been used to trace chiral molecular states and changes of protein configurations. In recent years, chiral plasmonic nanostructures have shown potential for applications ranging from pathogen sensing to novel optical materials. The plasmonic coupling of the individual elements of such metallic structures is a crucial prerequisite to obtain sizeable CD signals. We here identify and implement various coupling entities-chiral and achiral-to demonstrate chiral transfer over distances close to 100nm. The coupling is realized by an achiral nanosphere situated between a pair of gold nanorods that are arranged far apart but in a chiral fashion using DNA origami. The transmitter particle causes a strong enhancement of the CD response, the emergence of an additional chiral feature at the resonance frequency of the nanosphere, and a redshift of the longitudinal plasmonic resonance frequency of the nanorods. Matching numerical simulations elucidate the intricate chiral optical fields in complex architectures. Here, the authors experimentally demonstrate chiral transfer over large distances up to 100 nm. They realise the coupling with an achiral nanosphere situated between a pair of distant gold nanorods arranged in a chiral fashion using DNA origami, and observe enhanced circular dichroism signals.
    Publication_Year:: 2021
    DOI:: https://doi.org/10.1038/s41467-021-22289-8
  - [[DNA Origami-Enabled Plasmonic Sensing]]
    - DNA Origami-Enabled Plasmonic Sensing
    source_file:: Liedl.csv
    Authors:: Dass, M; Gur, FN; Kolqtaj, K; Urban, MJ; Liedl, T
    Source_Title:: JOURNAL OF PHYSICAL CHEMISTRY C
    IF:: 4.126
    Document_Type:: Article
    Abstract:: The reliable programmability of DNA origami makes it an extremely attractive tool for bottom-up self-assembly of complex nanostructures. Utilizing this property for the tuned arrangement of plasmonic nanoparticles holds great promise particularly in the field of biosensing. Plasmonic particles are beneficial for sensing in multiple ways, from enhancing fluorescence to enabling a visualization of the nanoscale dynamic actuation via chiral rearrangements. In this Perspective, we discuss the recent developments and possible future directions of DNA origami-enabled plasmonic sensing systems. We start by discussing recent advancements in the area of fluorescence-based plasmonic sensing using DNA origami. We then move on to surface-enhanced Raman spectroscopy sensors followed by chiral sensing, both utilizing DNA origami nanostructures. We conclude by providing our own views on the future prospects for plasmonic biosensors enabled using DNA origami.
    Publication_Year:: 2021
    DOI:: https://doi.org/10.1021/acs.jpcc.0c11238
  - [[DNA Origami Nano-Sheets and Nano-Rods Alter the Orientational Order in a Lyotropic Chromonic Liquid Crystal]]
    - DNA Origami Nano-Sheets and Nano-Rods Alter the Orientational Order in a Lyotropic Chromonic Liquid Crystal
    source_file:: Liedl.csv
    Authors:: Zhang, BR; Martens, K; Kneer, L; Funck, T; Nguyen, L; Berger, R; Dass, M; Kempter, S; Schmidtke, J; Liedl, T; Kitzerow, HS
    Source_Title:: NANOMATERIALS
    IF:: 5.076
    Document_Type:: Article
    Abstract:: Rod-like and sheet-like nano-particles made of desoxyribonucleic acid (DNA) fabricated by the DNA origami method (base sequence-controlled self-organized folding of DNA) are dispersed in a lyotropic chromonic liquid crystal made of an aqueous solution of disodium cromoglycate. The respective liquid crystalline nanodispersions are doped with a dichroic fluorescent dye and their orientational order parameter is studied by means of polarized fluorescence spectroscopy. The presence of the nano-particles is found to slightly reduce the orientational order parameter of the nematic mesophase. Nano-rods with a large length/width ratio tend to preserve the orientational order, while more compact stiff nano-rods and especially nano-sheets reduce the order parameter to a larger extent. In spite of the difference between the sizes of the DNA nano-particles and the rod-like columnar aggregates forming the liquid crystal, a similarity between the shapes of the former and the latter seems to be better compatible with the orientational order of the liquid crystal.
    Publication_Year:: 2020
    DOI:: https://doi.org/10.3390/nano10091695
  - [[DNA-Based Assembly of Quantum Dots into Dimers and Helices]]
    - DNA-Based Assembly of Quantum Dots into Dimers and Helices
    source_file:: Liedl.csv
    Authors:: Zhang, T; Liedl, T
    Source_Title:: NANOMATERIALS
    IF:: 5.076
    Document_Type:: Article
    Abstract:: Owing to their unique optical properties, colloidal quantum dots (QDs) have attracted much attention as versatile fluorescent markers with broad biological and physical applications. On the other hand, DNA-based assembly has proven to be a powerful bottom-up approach to create designer nanoscale objects and to use these objects for the site-directed arrangement of guest components. To achieve good colloidal stability and accurate positioning of QDs on DNA templates, robust QD surface functionalization is crucial. Here, we present a simple and reliable conjugation method for the direct attachment of DNA molecules to QDs. Phosphorothiolated regions of chimera oligonucleotides are attached and incorporated into a ZnS layer freshly growing in situ on QDs that were rendered water soluble with hydrophilic ligands in a prior step. The reaction can be completed in a 2 mL plastic tube without any special equipment. The utility of these DNA-labeled QDs is demonstrated via prototypical assemblies such as QDs dimers with various spacings and chiral helical architectures.
    Publication_Year:: 2019
    DOI:: https://doi.org/10.3390/nano9030339
  - [[Sensing Picomolar Concentrations of RNA Using Switchable Plasmonic Chirality]]
    - Sensing Picomolar Concentrations of RNA Using Switchable Plasmonic Chirality
    source_file:: Liedl.csv
    Authors:: Funck, T; Nicoli, F; Kuzyk, A; Liedl, T
    Source_Title:: ANGEWANDTE CHEMIE-INTERNATIONAL EDITION
    IF:: 15.336
    Document_Type:: Article
    Abstract:: Detecting small sequences of RNA in biological samples such as microRNA or viral RNA demands highly sensitive and specific methods. Here, a reconfigurable DNA origami template has been used where a chiral arrangement of gold nanorods on the structure can lead to the generation of strong circular dichroism (CD). Switching of the cross-like DNA structure is achieved by the addition of nucleic acid sequences, which arrests the structure in one of the possible chiral states by specific molecular recognition. A specific sequence can thus be detected through the resulting changes in the plasmonic CD spectrum. We show the sensitive and selective detection of a target RNA sequence from the hepatitis C virus genome. The RNA binds to a complementary sequence that is part of the lock mechanism, which leads to the formation of a defined state of the plasmonic system with a distinct optical response. With this approach, we were able to detect this specific RNA sequence at concentrations as low as 100 pm.
    Publication_Year:: 2018
    DOI:: https://doi.org/10.1002/anie.201807029
  - [[Circular Dichroism of Chiral Molecules in DNA-Assembled Plasmonic Hotspots]]
    - Circular Dichroism of Chiral Molecules in DNA-Assembled Plasmonic Hotspots
    source_file:: Liedl.csv
    Authors:: Kneer, LM; Roller, EM; Besteiro, LV; Schreiber, R; Govorov, AO; Liedl, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: The chiral state of a molecule plays a crucial role in molecular recognition and biochemical reactions. Because of this and owing to the fact that most modern drugs are chiral, the sensitive and reliable detection of the chirality of molecules is of great interest to drug development. The majority of naturally occurring biomolecules exhibit circular dichroism (CD) in the UV range. Theoretical studies and several experiments have demonstrated that this UV-CD can be transferred into the plasmonic frequency domain when metal surfaces and chiral biomolecules are in close proximity. Here, we demonstrate that the CD transfer effect can be drastically enhanced by placing chiral molecules, here double-stranded DNA, inside a plasmonic hotspot. By using different particle types (gold, silver, spheres, and rods) and by exploiting the versatility of DNA origami, we were able to systematically study the impact of varying particle distances on the CD transfer efficiency and to demonstrate CD transfer over the whole optical spectrum down to the near-infrared. For this purpose, nanorods were also placed upright on DNA origami sheets, forming strong optical antennas. Theoretical models, demonstrating the intricate relationships between molecular chirality and achiral electric fields, support our experimental findings. From both experimental measurements and theoretical considerations, we conclude that the transferred CD is most intensive for systems with strong plasmonic hotspots, as we find them in relatively small gaps (5-12 nm) between spherical nanoparticles and preferably between the tips of nanorods.
    Publication_Year:: 2018
    DOI:: https://doi.org/10.1021/acsnano.8b03146
  - [[3D DNA Origami Crystals]]
    - 3D DNA Origami Crystals
    source_file:: Liedl.csv
    Authors:: Zhang, T; Hartl, C; Frank, K; Heuer-Jungemann, A; Fischer, S; Nickels, PC; Nickel, B; Liedl, T
    Source_Title:: ADVANCED MATERIALS
    IF:: 30.849
    Document_Type:: Article
    Abstract:: 3D crystals assembled entirely from DNA provide a route to design materials on a molecular level and to arrange guest particles in predefined lattices. This requires design schemes that provide high rigidity and sufficiently large open guest space. A DNA-origami-based tensegrity triangle structure that assembles into a 3D rhombohedral crystalline lattice with an open structure in which 90% of the volume is empty space is presented here. Site-specific placement of gold nanoparticles within the lattice demonstrates that these crystals are spacious enough to efficiently host 20 nm particles in a cavity size of 1.83 x 10(5) nm(3), which would also suffice to accommodate ribosome-sized macromolecules. The accurate assembly of the DNA origami lattice itself, as well as the precise incorporation of gold particles, is validated by electron microscopy and small-angle X-ray scattering experiments. The results show that it is possible to create DNA building blocks that assemble into lattices with customized geometry. Site-specific hosting of nano objects in the optically transparent DNA lattice sets the stage for metamaterial and structural biology applications.
    Publication_Year:: 2018
    DOI:: https://doi.org/10.1002/adma.201800273
  - [[Position Accuracy of Gold Nanoparticles on DNA Origami Structures Studied with Small-Angle X-ray Scattering]]
    - Position Accuracy of Gold Nanoparticles on DNA Origami Structures Studied with Small-Angle X-ray Scattering
    source_file:: Liedl.csv
    Authors:: Hartl, C; Frank, K; Amenitsch, H; Fischer, S; Liedl, T; Nickel, B
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: DNA origami objects allow for accurate positioning of guest molecules in three dimensions. Validation and understanding of design strategies for particle attachment as well as analysis of specific particle arrangements are desirable. Small-angle X-ray scattering (SAXS) is suited to probe distances of nano-objects with subnanometer resolution at physiologically relevant conditions including pH and salt and at varying temperatures. Here, we show that the pair density distribution function (PDDF) obtained from an indirect Fourier transform of SAXS intensities in a model-free way allows to investigate prototypical DNA origami-mediated gold nanoparticle (AuNP) assemblies. We analyze the structure of three AuNP-dimers on a DNA origami block, an AuNP trimer constituted by those dimers, and a helical arrangement of nine AuNPs on a DNA origami cylinder. For the dimers, we compare the model-free PDDF and explicit modeling of the SAXS intensity data by superposition of scattering intensities of the scattering objects. The PDDF of the trimer is verified to be a superposition of its dimeric contributions, that is, here AuNP-DNA origami assemblies were used as test boards underlining the validity of the PDDF analysis beyond pairs of AuNPs. We obtain information about AuNP distances with an uncertainty margin of 1.2 nm. This readout accuracy in turn can be used for high precision placement of AuNP by careful design of the AuNP attachment sites on the DNA-structure and by fine-tuning of the connector types.
    Publication_Year:: 2018
    DOI:: https://doi.org/10.1021/acs.nanolett.8b00412
  - [[Sculpting light by arranging optical components with DNA nanostructures]]
    - Sculpting light by arranging optical components with DNA nanostructures
    source_file:: Liedl.csv
    Authors:: Pilo-Pais, M; Acuna, GP; Tinnefeld, P; Liedl, T
    Source_Title:: MRS BULLETIN
    IF:: 6.578
    Document_Type:: Article
    Abstract:: DNA nanotechnology has developed into a state where the design and assembly of complex nanoscale structures is fast, reliable, cost effective, and accessible to nonexperts. Nanometer-precise positioning of organic (e.g., dyes and biomolecules) and inorganic (e.g., metal nanoparticles and colloidal quantum dots) components on DNA nanostructures is straightforward and modular. In this article, we identify the opportunities and challenges that DNA-assembled devices and materials face for optical antennas, metamaterials, and sensing applications. With the ability to arrange hybrid components in defined geometries, plasmonic effects will, for example, amplify molecular recognition transduction such that single-molecule events will be measureable with simple devices. On a larger scale, DNA nanotechnology has the potential to break the symmetry of common self-assembled functional materials, creating predefined optical properties such as refractive-index tuning and topological insulation.
    Publication_Year:: 2017
    DOI:: https://doi.org/10.1557/mrs.2017.278
  - [[Directional Photonic Wire Mediated by Homo-Forster Resonance Energy Transfer on a DNA Origami Platform]]
    - Directional Photonic Wire Mediated by Homo-Forster Resonance Energy Transfer on a DNA Origami Platform
    source_file:: Liedl.csv
    Authors:: Nicoli, F; Barth, A; Bae, W; Neukirchinger, F; Crevenna, AH; Lamb, DC; Liedl, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: Elaborating efficient strategies and deepening the understanding of light transport at the nanoscale is of great importance for future designs of artificial light-harvesting assemblies and dye-based photonic circuits. In this work, we focus on studying the phenomenon of Forster resonance energy transfer (FRET) among fluorophores of the same kind (homo-FRET) and its implications for energy cascades containing two or three different dye molecules. Utilizing the spatial programmability of DNA origami, we arranged a chain of cyanine 3 (Cy3) dyes flanked at one end with a dye of lower excitation energy, cyanine 5 (Cy5), with or without an additional dye of higher excitation energy, Alexa488, at the other end. We characterized the response of our fluorophore assemblies with bulk and single-molecule spectroscopy and support our measurements by Monte Carlo modeling of energy transfer within the system. We find that, depending on the arrangement of the fluorophores, homo-FRET between the Cy3 dyes can lead to an overall enhanced energy transfer to the acceptor fluorophore. Furthermore, we systematically analyzed the homo-FRET system by addressing the fluorescence lifetime and anisotropy. Finally, we built a homo-FRET-mediated photonic wire capable of transferring energy through the homo-FRET system from the blue donor dye (Alexa488) to the red acceptor fluorophore (Cy5) across a total distance of 16 nm.
    Publication_Year:: 2017
    DOI:: https://doi.org/10.1021/acsnano.7b05631
  - [[Hotspot-mediated non-dissipative and ultrafast plasmon passage]]
    - Hotspot-mediated non-dissipative and ultrafast plasmon passage
    source_file:: Liedl.csv
    Authors:: Roller, EM; Besteiro, LV; Pupp, C; Khorashad, LK; Govorov, AO; Liedl, T
    Source_Title:: NATURE PHYSICS
    IF:: 20.034
    Document_Type:: Article
    Abstract:: Plasmonic nanoparticles hold great promise as photonhandling elements and as channels for coherent transfer of energy and information in future all-optical computing devices(1-5). Coherent energy oscillations between two spatially separated plasmonic entities via a virtual middle state exemplify electron-based population transfer, but their realization requires precise nanoscale positioning of heterogeneous particles(6-10). Here, we show the assembly and optical analysis of a triple-particle system consisting of two gold nanoparticles with an interspaced silver island. We observe strong plasmonic coupling between the spatially separated gold particles, mediated by the connecting silver particle, with almost no dissipation of energy. As the excitation energy of the silver island exceeds that of the gold particles, only quasi-occupation of the silver transfer channel is possible. We describe this effect both with exact classical electrodynamic modelling and qualitative quantum-mechanical calculations. We identify the formation of strong hotspots between all particles as the main mechanism for the lossless coupling and thus coherent ultrafast energy transfer between the remote partners. Our findings could prove useful for quantum gate operations, as well as for classical charge and information transfer processes.
    Publication_Year:: 2017
    DOI:: https://doi.org/10.1038/NPHYS4120
  - [[Self-Assembled DNA Tubes Forming Helices of Controlled Diameter and Chirality]]
    - Self-Assembled DNA Tubes Forming Helices of Controlled Diameter and Chirality
    source_file:: Liedl.csv
    Authors:: Maier, AM; Bae, W; Schiffels, D; Emmerig, JF; Schiff, M; Liedl, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: Multihelical DNA bundles could enhance the functionality of nanomaterials and serve as model architectures to mimic protein filaments on the molecular and cellular level. We report the self-assembly of micrometer-sized helical DNA nanotubes with widely controllable helical diameters ranging from tens of nanometers to a few micrometers. Nanoscale helical shapes of DNA tile tubes (4-, 6-, 8-, 10-, and 12-helix tile tubes) are achieved by introducing discrete amounts of bending and twist through base pair insertions and/or deletions. Microscale helical diameters, which require smaller amounts of twist and bending, are achieved by controlling the intrinsic supertwist present in tile tubes with uneven number of helices (11-, 13-, and 15-helix tile tubes). Supertwist fine-tuning also allows us to produce helical nanotubes of defined chirality.
    Publication_Year:: 2017
    DOI:: https://doi.org/10.1021/acsnano.6b05602
  - [[Proximity-Induced H-Aggregation of Cyanine Dyes on DNA-Duplexes]]
    - Proximity-Induced H-Aggregation of Cyanine Dyes on DNA-Duplexes
    source_file:: Liedl.csv
    Authors:: Nicoli, F; Roos, MK; Hemmig, EA; Di Antonio, M; de Vivie-Riedle, R; Liedl, T
    Source_Title:: JOURNAL OF PHYSICAL CHEMISTRY A
    IF:: 2.781
    Document_Type:: Article
    Abstract:: A wide variety of organic dyes form, under certain conditions, clusters know as J- and H-aggregates. Cyanine dyes are such a class of molecules where the spatial proximity of several dyes leads to overlapping electron orbitals and thus to the creation of a new energy landscape compared to, that of the individual units. In this work, We create artificial H-aggregates of exactly two Cyanine 3 (Cy3) dyes by covalently linking them to a DNA molecule with controlled subnanometer distances. The absorption spectra of these coupled systems exhibit a blue-shifted peak, whose intensity varies depending on the distance between the dyes and the rigidity of the DNA template. Simulated vibrational resolved spectra, based on molecular orbital theory, excellently reproduce the experimentally observed features. Circular dichroism spectroscopy additionally reveals distinct signals, which indicates a chiral arrangement of the dye molecules. Molecular dynamic simulations of a Cy3-Cy3 construct including a 14-base pair DNA sequence verified chiral stacking of the dye molecules.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1021/acs.jpca.6b10939
  - [[Quantitative Single-Molecule Surface Enhanced Raman Scattering by Optothermal Tuning of DNA Origami-Assembled Plasmonic Nanoantennas]]
    - Quantitative Single-Molecule Surface Enhanced Raman Scattering by Optothermal Tuning of DNA Origami-Assembled Plasmonic Nanoantennas
    source_file:: Liedl.csv
    Authors:: Simoncelli, S; Roller, EM; Urban, P; Schreiber, R; Turberfield, AJ; Liedl, T; Lohmuller, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: DNA origami is a powerful approach for assembling plasmonic nanoparticle dimers and Raman dyes with high yields and excellent positioning control. Here we show how optothermal-induced shrinking of a DNA origami template can be employed to control the gap sizes between two 40 nm gold nanoparticles in a range from 1 to 2 nm. The high field confinement achieved with this optothermal approach was demonstrated by detection of surface enhanced Raman spectroscopy (SERS) signals from single molecules that are precisely placed within the DNA origami template that spans the nanoparticle gap. By comparing the SERS intensity with respect to the field enhancement in the plasmonic hot-spot region, we found good agreement between measurement and theory. Our straightforward approach for the fabrication of addressable plasmonic nanosensors by DNA origami demonstrates a path toward future sensing applications with single-molecule resolution.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1021/acsnano.6b05276
  - [[Molecular force spectroscopy with a DNA origami-based nanoscopic force clamp]]
    - Molecular force spectroscopy with a DNA origami-based nanoscopic force clamp
    source_file:: Liedl.csv
    Authors:: Nickels, PC; Wunsch, B; Holzmeister, P; Bae, W; Kneer, LM; Grohmann, D; Tinnefeld, P; Liedl, T
    Source_Title:: SCIENCE
    IF:: 47.728
    Document_Type:: Article
    Abstract:: Forces in biological systems are typically investigated at the single-molecule level with atomic force microscopy or optical and magnetic tweezers, but these techniques suffer from limited data throughput and their requirement for a physical connection to the macroscopic world. We introduce a self-assembled nanoscopic force clamp built from DNA that operates autonomously and allows massive parallelization. Single-stranded DNA sections of an origami structure acted as entropic springs and exerted controlled tension in the low piconewton range on a molecular system, whose conformational transitions were monitored by single-molecule Frster resonance energy transfer. We used the conformer switching of a Holliday junction as a benchmark and studied the TATA-binding protein-induced bending of a DNA duplex under tension. The observed suppression of bending above 10 piconewtons provides further evidence of mechanosensitivity in gene regulation.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1126/science.aah5974
  - [[Shape and Interhelical Spacing of DNA Origami Nanostructures Studied by Small-Angle X-ray Scattering]]
    - Shape and Interhelical Spacing of DNA Origami Nanostructures Studied by Small-Angle X-ray Scattering
    source_file:: Liedl.csv
    Authors:: Fischer, S; Hartl, C; Frank, K; Radler, JO; Liedl, T; Nickel, B
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: Scaffolded DNA origami nanostructures enable the self-assembly of arbitrarily shaped objects with unprecedented accuracy. Yet, varying physiological conditions are prone to induce slight structural changes in the nanoscale architecture. Here, we report on high precision measurements of overall shape and interhelical distance of three prototypic DNA origami structures in solution using synchrotron small angle X-ray scattering. Sheet-, brick-, and cylinder-shaped DNA constructs were assembled and the shape factors determined with angstrom resolution from fits to the scattering profiles. With decreasing MgCl2 concentration electrostatic swelling of both shape cross section and interhelical DNA spacing of the DNA origami structures is observed. The structures tolerate up to 10% interhelical expansion before they disintegrate. In contrast, with increasing temperature, the cylinder-shaped structures show no thermal expansion in a wide temperature window before they abruptly melt above 50 degrees C. Details on molecular structure of DNA origami can also be obtained using in-house X-ray scattering equipment and, hence, allow for routine folding and stability testing of DNA-based agents that are designed to operate under varying salt conditions.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1021/acs.nanolett.6b01335
  - [[DNA Origami Seesaws as Comparative Binding Assay]]
    - DNA Origami Seesaws as Comparative Binding Assay
    source_file:: Liedl.csv
    Authors:: Nickels, PC; Hoiberg, HC; Simmel, SS; Holzmeister, P; Tinnefeld, P; Liedl, T
    Source_Title:: CHEMBIOCHEM
    IF:: 3.164
    Document_Type:: Article
    Abstract:: The application of commonly used force spectroscopy in biological systems is often limited by the need for an invasive tether connecting the molecules of interest to a bead or cantilever tip. Here we present a DNA origami-based prototype in a comparative binding assay. It has the advantage of in situ readout without any physical connection to the macroscopic world. The seesaw-like structure has a lever that is able to move freely relative to its base. Binding partners on each side force the structure into discrete and distinguishable conformations. Model experiments with competing DNA hybridisation reactions yielded a drastic shift towards the conformation with the stronger binding interaction. With reference DNA duplexes of tuneable length on one side, this device can be used to measure ligand interactions in comparative assays.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1002/cbic.201600059
  - [[Magnetic Propulsion of Microswimmers with DNA-Based Flagellar Bundles]]
    - Magnetic Propulsion of Microswimmers with DNA-Based Flagellar Bundles
    source_file:: Liedl.csv
    Authors:: Maier, AM; Weig, C; Oswald, P; Frey, E; Fischer, P; Liedl, T
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: We show that DNA-based self-assembly can serve as a general and flexible tool to construct artificial flagella of several micrometers in length and only tens of nanometers in diameter. By attaching the DNA flagella to biocompatible magnetic microparticles, we provide a proof of concept demonstration of hybrid structures that, when rotated in an external magnetic field, propel by means of a flagellar bundle, similar to self-propelling peritrichous bacteria. Our theoretical analysis predicts that flagellar bundles that possess a length-dependent bending stiffness should exhibit a superior swimming speed compared to swimmers with a single appendage. The DNA self-assembly method permits the realization of these improved flagellar bundles in good agreement with our quantitative model. DNA flagella with well-controlled shape could fundamentally increase the functionality of fully biocompatible nanorobots and extend the scope and complexity of active materials.
    Publication_Year:: 2016
    DOI:: https://doi.org/10.1021/acs.nanolett.5b03716
  - [[DNA-Based Self-Assembly of Fluorescent Nanodiamonds]]
    - DNA-Based Self-Assembly of Fluorescent Nanodiamonds
    source_file:: Liedl.csv
    Authors:: Zhang, T; Neumann, A; Lindlau, J; Wu, YZ; Prarnanik, G; Naydenov, B; Jelezko, F; Schuder, F; Huber, S; Huber, M; Stehr, F; Hogele, A; Weil, T; Liedl, T
    Source_Title:: JOURNAL OF THE AMERICAN CHEMICAL SOCIETY
    IF:: 15.419
    Document_Type:: Article
    Abstract:: As a step toward deterministic and scalable assembly of ordered spin arrays we here demonstrate a bottom-up approach to position fluorescent nanodiamonds (NDs) with nanometer precision on DNA origami structures. We have realized a reliable and broadly applicable surface modification strategy that results in DNA-functionalized and perfectly dispersed NDs that were then self-assembled in predefined geometries. With optical studies we show that the fluorescence properties of the nitrogen-vacancy color centers in NDs are preserved during surface modification and DNA assembly. As this method allows the nanoscale arrangement of fluorescent NDs together with other optically active components in complex geometries, applications based on self-assembled spin lattices or plasmon-enhanced spin sensors as well as improved fluorescent labeling for bioimaging could be envisioned.
    Publication_Year:: 2015
    DOI:: https://doi.org/10.1021/jacs.5b04857
  - [[DNA-linked superlattices get into shape]]
    - DNA-linked superlattices get into shape
    source_file:: Liedl.csv
    Authors:: Nickel, B; Liedl, T
    Source_Title:: NATURE MATERIALS
    IF:: 43.841
    Document_Type:: Article
    Abstract:: 
    Publication_Year:: 2015
    DOI:: https://doi.org/10.1038/nmat4376
  - [[Membrane-Assisted Growth of DNA Origami Nanostructure Arrays]]
    - Membrane-Assisted Growth of DNA Origami Nanostructure Arrays
    source_file:: Liedl.csv
    Authors:: Kocabey, S; Kempter, S; List, J; Xing, YZ; Bae, W; Schiffels, D; Shih, WM; Simmel, FC; Liedl, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: Biological membranes fulfill many important tasks within living organisms. In addition to separating cellular volumes, membranes confine the space available to membrane-associated proteins to two dimensions (2D), which greatly increases their probability to interact with each other and assemble into multiprotein complexes. We here employed two DNA origami structures functionalized with cholesterol moieties as membrane anchors-a three-layered rectangular block and a Y-shaped DNA structure to mimic membrane-assisted assembly into hierarchical superstructures on supported lipid bilayers and small unilamellar vesicles. As designed, the DNA constructs adhered to the lipid bilayers mediated by the cholesterol anchors and diffused freely in 2D with diffusion coefficients depending on their size and number of cholesterol modifications. Different sets of multimerization oligonucleotides added to bilayer-bound origami block structures induced the growth of either linear polymers or two-dimensional lattices on the membrane. Y-shaped DNA origami structures associated into triskelion homotrimers and further assembled into weakly ordered arrays of-hexagons and pentagons, which resembled the geometry of clathrin-coated pits. Our results demonstrate the potential to realize artificial self-assembling systems that mimic the hierarchical formation of polyhedral lattices on cytoplasmic membranes.
    Publication_Year:: 2015
    DOI:: https://doi.org/10.1021/acsnano.5b00161
  - [[Cellular Uptake of Tile-Assembled DNA Nanotubes]]
    - Cellular Uptake of Tile-Assembled DNA Nanotubes
    source_file:: Liedl.csv
    Authors:: Kocabey, S; Meinl, H; MacPherson, IS; Cassinelli, V; Manetto, A; Rothenfusser, S; Liedl, T; Lichtenegger, FS
    Source_Title:: NANOMATERIALS
    IF:: 5.076
    Document_Type:: Article
    Abstract:: DNA-based nanostructures have received great attention as molecular vehicles for cellular delivery of biomolecules and cancer drugs. Here, we report on the cellular uptake of tubule-like DNA tile-assembled nanostructures 27 nm in length and 8 nm in diameter that carry siRNA molecules, folic acid and fluorescent dyes. In our observations, the DNA structures are delivered to the endosome and do not reach the cytosol of the GFP-expressing HeLa cells that were used in the experiments. Consistent with this observation, no elevated silencing of the GFP gene could be detected. Furthermore, the presence of up to six molecules of folic acid on the carrier surface did not alter the uptake behavior and gene silencing. We further observed several challenges that have to be considered when performing in vitro and in vivo experiments with DNA structures: (i) DNA tile tubes consisting of 42 nt-long oligonucleotides and carrying single- or double-stranded extensions degrade within one hour in cell medium at 37 degrees C, while the same tubes without extensions are stable for up to eight hours. The degradation is caused mainly by the low concentration of divalent ions in the media. The lifetime in cell medium can be increased drastically by employing DNA tiles that are 84 nt long. (ii) Dyes may get cleaved from the oligonucleotides and then accumulate inside the cell close to the mitochondria, which can lead to misinterpretation of data generated by flow cytometry and fluorescence microscopy. (iii) Single-stranded DNA carrying fluorescent dyes are internalized at similar levels as the DNA tile-assembled tubes used here.
    Publication_Year:: 2015
    DOI:: https://doi.org/10.3390/nano5010047
  - [[DNA-Assembled Nanoparticle Rings Exhibit Electric and Magnetic Resonances at Visible Frequencies]]
    - DNA-Assembled Nanoparticle Rings Exhibit Electric and Magnetic Resonances at Visible Frequencies
    source_file:: Liedl.csv
    Authors:: Roller, EM; Khorashad, LK; Fedoruk, M; Schreiber, R; Govorov, AO; Liedl, T
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: Metallic nanostructures can be used to manipulate light on the subwavelength scale to create tailored optical material properties. Next to electric responses, artificial optical magnetism is of particular interest but difficult to achieve at visible wavelengths. DNA-self-assembly has proved to serve as a viable method to template plasmonic materials with nanometer precision and to produce large quantities of metallic objects with high yields. We present here the fabrication of self-assembled ring-shaped plasmonic metamolecules that are composed of four to eight single metal nanoparticles with full stoichiometric and geometric control. Scattering spectra of single rings as well as absorption spectra of solutions containing the metamolecules are used to examine the unique plasmonic features, which are compared to computational simulations. We demonstrate that the electric and magnetic plasmon resonance modes strongly correlate with the exact shape of the structures. In particular, our computations reveal the magnetic plasmons only for particle rings of broken symmetries, which is consistent with our experimental data. We stress the feasibility of DNA self-assembly as a method to create bulk plasmonic materials and metamolecules that may be applied as building blocks in plasmonic devices.
    Publication_Year:: 2015
    DOI:: https://doi.org/10.1021/nl5046473
  - [[DNA Origami Structures Directly Assembled from Intact Bacteriophages]]
    - DNA Origami Structures Directly Assembled from Intact Bacteriophages
    source_file:: Liedl.csv
    Authors:: Nickels, PC; Ke, YG; Jungmann, R; Smith, DM; Leichsenring, M; Shih, WM; Liedl, T; Hogberg, B
    Source_Title:: SMALL
    IF:: 13.281
    Document_Type:: Article
    Abstract:: 
    Publication_Year:: 2014
    DOI:: https://doi.org/10.1002/smll.201303442
  - [[Hierarchical assembly of metal nanoparticles, quantum dots and organic dyes using DNA origami scaffolds]]
    - Hierarchical assembly of metal nanoparticles, quantum dots and organic dyes using DNA origami scaffolds
    source_file:: Liedl.csv
    Authors:: Schreiber, R; Do, J; Roller, EM; Zhang, T; Schuller, VJ; Nickels, PC; Feldmann, J; Liedl, T
    Source_Title:: NATURE NANOTECHNOLOGY
    IF:: 39.213
    Document_Type:: Article
    Abstract:: The self-assembly of nanoscale elements into three-dimensional structures with precise shapes and sizes is important in fields such as nanophotonics, metamaterials and biotechnology(1,2). Short molecular linkers have previously been used to create assemblies of nanoparticles(3-9), but the approach is limited to small interparticle distances, typically less than 10 nm. Alternatively, DNA origami(10,11) can precisely organize nanoscale objects over much larger length scales. Here we show that rigid DNA origami scaffolds can be used to assemble metal nanoparticles, quantum dots and organic dyes into hierarchical nanoclusters that have a planet-satellite-type structure. The nanoclusters have a tunable stoichiometry, defined distances of 5-200 nm between components, and controllable overall sizes of up to 500 nm. We also show that the nanoscale components can be positioned along the radial DNA spacers of the nanostructures, which allows short- and long-range interactions between nanoparticles and dyes to be studied in solution. The approach could, in the future, be used to construct efficient energy funnels, complex plasmonic architectures, and porous, nanoengineered scaffolds for catalysis.
    Publication_Year:: 2014
    DOI:: https://doi.org/10.1038/NNANO.2013.253
  - [[Chiral plasmonic DNA nanostructures with switchable circular dichroism]]
    - Chiral plasmonic DNA nanostructures with switchable circular dichroism
    source_file:: Liedl.csv
    Authors:: Schreiber, R; Luong, N; Fan, ZY; Kuzyk, A; Nickels, PC; Zhang, T; Smith, DM; Yurke, B; Kuang, W; Govorov, AO; Liedl, T
    Source_Title:: NATURE COMMUNICATIONS
    IF:: 14.919
    Document_Type:: Article
    Abstract:: Circular dichroism spectra of naturally occurring molecules and also of synthetic chiral arrangements of plasmonic particles often exhibit characteristic bisignate shapes. Such spectra consist of peaks next to dips (or vice versa) and result from the superposition of signals originating from many individual chiral objects oriented randomly in solution. Here we show that by first aligning and then toggling the orientation of DNA-origami-scaffolded nanoparticle helices attached to a substrate, we are able to reversibly switch the optical response between two distinct circular dichroism spectra corresponding to either perpendicular or parallel helix orientation with respect to the light beam. The observed directional circular dichroism of our switchable plasmonic material is in good agreement with predictions based on dipole approximation theory. Such dynamic metamaterials introduce functionality into soft matter-based optical devices and may enable novel data storage schemes or signal modulators.
    Publication_Year:: 2013
    DOI:: https://doi.org/10.1038/ncomms3948
  - [[M1.3 - a small scaffold for DNA origami]]
    - M1.3 - a small scaffold for DNA origami
    source_file:: Liedl.csv
    Authors:: Said, H; Schuller, VJ; Eber, FJ; Wege, C; Liedl, T; Richert, C
    Source_Title:: NANOSCALE
    IF:: 7.79
    Document_Type:: Article
    Abstract:: The DNA origami method produces programmable nanoscale objects that form when one long scaffold strand hybridizes to numerous oligonucleotide staple strands. One scaffold strand is dominating the field: M13mp18, a bacteriophage-derived vector 7249 nucleotides in length. The full-length M13 is typically folded by using over 200 staple oligonucleotides. Here we report the convenient preparation of a 704 nt fragment dubbed M1.3 as a linear or cyclic scaffold and the assembly of small origami structures with just 15-24 staple strands. A typical M1.3 origami is large enough to be visualized by TEM, but small enough to show a cooperativity in its assembly and thermal denaturation that is reminiscent of oligonucleotide duplexes. Due to its medium size, M1.3 origami with globally modified staples is affordable. As a proof of principle, two origami structures with globally 5'-capped staples were prepared and were shown to give higher UV-melting points than the corresponding assembly with unmodified DNA. M1.3 has the size of a gene, not a genome, and may function as a model for gene-based nanostructures. Small origami with M1.3 as a scaffold may serve as a workbench for chemical, physical, and biological experiments.
    Publication_Year:: 2013
    DOI:: https://doi.org/10.1039/c2nr32393a
  - [[DNA-based self-assembly of chiral plasmonic nanostructures with tailored optical response]]
    - DNA-based self-assembly of chiral plasmonic nanostructures with tailored optical response
    source_file:: Liedl.csv
    Authors:: Kuzyk, A; Schreiber, R; Fan, ZY; Pardatscher, G; Roller, EM; Hogele, A; Simmel, FC; Govorov, AO; Liedl, T
    Source_Title:: NATURE
    IF:: 49.962
    Document_Type:: Article
    Abstract:: Matter structured on a length scale comparable to or smaller than the wavelength of light can exhibit unusual optical properties(1). Particularly promising components for such materials are metal nanostructures, where structural alterations provide a straightforward means of tailoring their surface plasmon resonances and hence their interaction with light(2,3). But the top-down fabrication of plasmonic materials with controlled optical responses in the visible spectral range remains challenging, because lithographic methods are limited in resolution and in their ability to generate genuinely three-dimensional architectures(4,5). Molecular self-assembly(6,7) provides an alternative bottom-up fabrication route not restricted by these limitations, and DNA-and peptide-directed assembly have proved to be viable methods for the controlled arrangement of metal nanoparticles in complex and also chiral geometries(8-14). Here we show that DNA origami(15,16) enables the high-yield production of plasmonic structures that contain nanoparticles arranged in nanometre-scale helices. We find, in agreement with theoretical predictions(17), that the structures in solution exhibit defined circular dichroism and optical rotatory dispersion effects at visible wavelengths that originate from the collective plasmon-plasmon interactions of the nanoparticles positioned with an accuracy better than two nanometres. Circular dichroism effects in the visible part of the spectrum have been achieved by exploiting the chiral morphology of organic molecules and the plasmonic properties of nanoparticles(18-20), or even without precise control over the spatial configuration of the nanoparticles(12,21,22). In contrast, the optical response of our nanoparticle assemblies is rationally designed and tunable in handedness, colour and intensity-in accordance with our theoretical model.
    Publication_Year:: 2012
    DOI:: https://doi.org/10.1038/nature10889
  - [[DNA Origami Nanopores]]
    - DNA Origami Nanopores
    source_file:: Liedl.csv
    Authors:: Bell, NAW; Engst, CR; Ablay, M; Divitini, G; Ducati, C; Liedl, T; Keyser, UF
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: We demonstrate the assembly of functional hybrid nanopores for single molecule sensing by inserting DNA origami structures into solid-state nanopores. In our experiments, single artificial nanopores based on DNA origami are repeatedly inserted in and ejected from solid-state nanopores with diameters around 15 nm. We show that these hybrid nanopores can be employed for the detection of A-DNA molecules. Our approach paves the way for future development of adaptable single-molecule nanopore sensors based on the combination of solid-state nanopores and DNA self-assembly.
    Publication_Year:: 2012
    DOI:: https://doi.org/10.1021/nl204098n
  - [[Cellular Immunostimulation by CpG-Sequence-Coated DNA Origami Structures]]
    - Cellular Immunostimulation by CpG-Sequence-Coated DNA Origami Structures
    source_file:: Liedl.csv
    Authors:: Schuller, VJ; Heidegger, S; Sandholzer, N; Nickels, PC; Suhartha, NA; Endres, S; Bourquin, C; Liedl, T
    Source_Title:: ACS NANO
    IF:: 15.881
    Document_Type:: Article
    Abstract:: To investigate the potential of DNA origami constructs as programmable and noncytotoxic immunostimulants, we tested the Immune responses induced by hollow 30-helix DNA origami tubes covered with up to 62 cytosine-phosphate-guanine (CpG) sequences in freshly isolated spleen cells. Unmethylated CpG sequences that are highly specific for bacterial DNA are recognized by a specialized receptor of the innate immune system localized in the endosome, the Toll-like receptor 9 (TLR9). When incubated with oligonucleotides containing CpGs, immune cells are stimulated through TLR9 to produce and secrete cytokine mediators such as interleukin-6 (IL-6) and interleukin-12p70 (IL-12p70), a process associated with the initiation of an immune response. In our studies, the DNA origami tube built from an 8634 nt long variant of the commonly used single-stranded DNA origami scaffold M13mp18 and 227 staple oligonucleotides decorated with 62 CpG-containing oligonucleotides triggered a strong immune response, characterized by cytokine production and immune cell activation, which was entirely dependent on TLR9 stimulation. Such decorated origami tubes also triggered higher immunostimulation than equal amounts of CpG oligonucleotides associated with a standard carrier system such as Lipofectamine. In the absence of CpG oligonucleotides, cytokine production induced by the origami tubes was low and was not related to TLR9 recognition. Fluorescent microscopy revealed localization of CpG-containing DNA origami structures in the endosome. The DNA constructs showed in contrast to Lipofectamine no detectable toxicity and did not affect the viability of splenocytes. We thus demonstrate that DNA origami constructs represent a delivery system for CpG oligonucleotides that Is both efficient and nontoxic.
    Publication_Year:: 2011
    DOI:: https://doi.org/10.1021/nn203161y
  - [[Direct Mechanical Measurements Reveal the Material Properties of Three-Dimensional DNA Origami]]
    - Direct Mechanical Measurements Reveal the Material Properties of Three-Dimensional DNA Origami
    source_file:: Liedl.csv
    Authors:: Kauert, DJ; Kurth, T; Liedl, T; Seidel, R
    Source_Title:: NANO LETTERS
    IF:: 11.189
    Document_Type:: Article
    Abstract:: The application of three-dimensional DNA origami objects as rigid mechanical mediators or force sensing elements requires detailed knowledge about their complex mechanical properties. Using magnetic tweezers, we directly measure the bending and torsional rigidities of four- and six-helix bundles assembled by this technique. Compared to duplex DNA, we find the bending rigidities to be greatly increased while the torsional rigidities are only moderately augmented. We present a mechanical model explicitly including the crossovers between the individual helices in the origami structure that reproduces the experimentally observed behavior. Our results provide an important basis for the future application of 3D DNA origami in nanomechanics.
    Publication_Year:: 2011
    DOI:: https://doi.org/10.1021/nl203503s
  - [[DNA Origami-Templated Growth of Arbitrarily Shaped Metal Nanoparticles]]
    - DNA Origami-Templated Growth of Arbitrarily Shaped Metal Nanoparticles
    source_file:: Liedl.csv
    Authors:: Schreiber, R; Kempter, S; Holler, S; Schuller, V; Schiffels, D; Simmel, SS; Nickels, PC; Liedl, T
    Source_Title:: SMALL
    IF:: 13.281
    Document_Type:: Article
    Abstract:: 
    Publication_Year:: 2011
    DOI:: https://doi.org/10.1002/smll.201100465
  - [[Single-Molecule FRET Ruler Based on Rigid DNA Origami Blocks]]
    - Single-Molecule FRET Ruler Based on Rigid DNA Origami Blocks
    source_file:: Liedl.csv
    Authors:: Stein, IH; Schuller, V; Bohm, P; Tinnefeld, P; Liedl, T
    Source_Title:: CHEMPHYSCHEM
    IF:: 3.102
    Document_Type:: Article
    Abstract:: Fluorescence resonance energy transfer (FRET) has become a work-horse for distance measurements on the nanometer scale and between single molecules. Recent model systems for the FRET distance dependence such as polyprolines and dsDNA suffered from limited persistence lengths and sample heterogeneity. We designed a series of rigid DNA origami blocks where each block is labeled with one donor and one acceptor at distances ranging between 2.5 and 14 nm. Since all dyes are attached in one plane to the top surface of the origami block, static effects of linker lengths cancel out in contrast to commonly used dsDNA. We used single-molecule spectroscopy to compare the origami-based ruler to dsDNA and found that the origami blocks directly yield the expected distance dependence of energy transfer since the influence of the linkers on the donor-acceptor distance is significantly reduced. Based on a simple geometric model for the inter-dye distances on the origami block, the Forster radius R-0 could directly be determined from the distance dependence of energy transfer yielding R-0 = 5.3 +/- 0.3 nm for the Cy3-Cy5 pair.
    Publication_Year:: 2011
    DOI:: https://doi.org/10.1002/cphc.201000781
- query-table:: true
  #+BEGIN_QUERY
  {:title [:h2 "List"]
   :query [:find (pull ?b [*])
           :where
           [?b :block/properties ?p]
           [(get ?p :source-file) ?t]
           [(= "Liedl.csv" ?t)]]}
  #+END_QUERY
